Dependencies/ShaderFixes/uscv.hlsl

```diff
@@ -0,0 +1,28 @@
+void main(
```

Comment:
I'll need a header comment for this file.  

We want to avoid mystery code as much as possible.  And this is all a mystery to me.

Header comments can explain *why* the code exists.  What it does can be determined from the code, but the key "why does it exist" needs a comment.